workflow-trace-viewer/build.gradle.kts

```diff
@@ -9,6 +11,8 @@ plugins {
 kotlin {
   jvm()
 
+  jvmToolchain(11)
```

Comment:
Why is this needed now? Something about the published artifact?

Reply:
This was actually a byproduct of pulling in telescope.